app/src/docker/Dockerfile

```
@@ -3,6 +3,7 @@ ARG JAR_FILE
 ARG ENTRYPOINT_FILE
 WORKDIR /project
 COPY ${ENTRYPOINT_FILE} entrypoint.sh
+RUN chmod 777 /project/build/tomcat
 RUN chmod +x entrypoint.sh
 COPY ${JAR_FILE} starter-java.jar
```

Make RUNs non-consecutive and add `mkdir -p /project/build/tomcat`
```suggestion
RUN chmod +x entrypoint.sh
COPY ${JAR_FILE} starter-java.jar
RUN mkdir -p /project/build/tomcat && chmod 777 /project/build/tomcat
```